mix.exs

```diff
@@ -92,16 +93,18 @@ defmodule Spawn.MixProject do
       # Statestores deps
       {:spawn_statestores_mssql, path: "./spawn_statestores/statestores_mssql", optional: false},
       {:spawn_statestores_mysql, path: "./spawn_statestores/statestores_mysql", optional: false},
-      {:spawn_statestores_postgres, path: "./spawn_statestores/statestores_postgres", optional: false},
-      {:spawn_statestores_sqlite, path: "./spawn_statestores/statestores_sqlite", optional: false},
-      {:spawn_statestores_cockroachdb, path: "./spawn_statestores/statestores_cockroachdb", optional: false},
+      {:spawn_statestores_postgres,
```

Comment:
disable formatting for this file

Reply:
Since that's not the main goal here, I think we can address that in another PR.